package.json

```diff
@@ -85,7 +85,7 @@
     "@grafana/data": "^11.1.1",
     "@grafana/lezer-logql": "^0.2.6",
     "@grafana/runtime": "^11.1.1",
-    "@grafana/scenes": "5.7.2",
+    "@grafana/scenes": "5.9.0",
```

Comment:
did we bump this to get the ability to run scene queries manually? I don't think we need that anymore since we don't need to wait on labels anymore, but maybe we still want to upgrade?

Reply:
I think we still need to run the service queries manually so we can update the `SERVICE_NAME_VAR` before the query runs if the time range is too long, otherwise we'd change the variable after the initial query was already run (and then immediately cancel the request)